### DISCUSSION TITLE
Face - EYES ONLY

### DISCUSSION THREAD

**Discussion by TOKYOJAB** [Mikubill/sd-webui-controlnet#944] [Ideas]
I do a lot of overrides where I replace the underlying video with Ironman, or a dog to a polar bear nonsense.
The face openpose is a fantastic addition but would really like an option to ONLY track the eyes and not the rest of the face.
For example if I was overlaying spiderman costume, alien.  or iron man then the ai would know where to line up the eyes but wouldn't try and make a human face.
A bit niche but would be nice.

![04203-2386918566-An alien creature, dark scifi, dark space ship corridor bokek, subsurface skin effects, bokeh,  f1 2 lens, incredible detail, re](https://user-images.githubusercontent.com/70209681/233553342-ef2e733e-5a3c-47ae-a271-98bdd362f943.jpg)

Ps. I have a method for Temporal Consistency with video keyframes, so it would be great with that. [Example](https://www.reddit.com/r/StableDiffusion/comments/12tuak6/1899_i_quite_like_the_dark_vibe_of_the_series)

**Comment by marcsyp** [comment]
I would very much appreciate this functionality, and had a really strong desire to have this work for a recent project where I had a lineart sketch with a specific eye position that I really wanted replicated, but always got eyes looking at the camera.  Do you have any workflows for matching eye position with CN??

**Reply by TOKYOJAB** [reply]
I found that only close up faces really capture the eye direction especially when I drag the controlnet resulotion to maximum. 
Also sometimes you can use photoshop or something to put two bright spots on the eyes of the photo that goes into controlnet or
 Let controlnet give you it's preview of the preprocessor and then take this into photoshop and add some extra indication of the eye position (then bring it back into controlnet with the preprocesser set to none)
 
 If all else fails the neural filters in photoshop are pretty good at turning the eyes (It's about the only thing they do well).
 
![image](https://user-images.githubusercontent.com/70209681/234488847-3a71a8c1-7337-4a40-90db-3d116c275458.png)
